packages/base/src/SearchBar/SearchBar-android.tsx

```diff
@@ -50,6 +50,7 @@ export class SearchBarAndroid extends Component<
   input!: TextInput;
 
   static defaultProps = {
+    lightTheme: false,
```

Comment:
lets remove `lightTheme` from each variant & keep background as theme.colors.searchBg as you did

Reply:
I have checked that if we remove the lightTheme the SearchBar will be dark

Reply:
please check this https://github.com/react-native-elements/react-native-elements/pull/3470/commits/e52099bf49fe08a052d9669a089533c7aee40194 commit, can we use separate colors for background Color as per platform?